cub/benchmarks/bench/scan/exclusive/by_key.cu

```diff
@@ -82,7 +84,7 @@ static void scan(nvbench::state& state, nvbench::type_list<KeyT, ValueT, OffsetT
   using val_input_it_t  = const ValueT*;
   using val_output_it_t = ValueT*;
   using equality_op_t   = ::cuda::std::equal_to<>;
-  using offset_t        = OffsetT;
+  using offset_t        = ::cuda::std::make_unsigned_t<OffsetT>;
```

Comment:
Why wouldn't we use `cub::detail::choose_offset_t` here, as we do for other benchmarks where the same is applicable? I think we'd prefer to promote small offset types to `uint32_t`, right?
```suggestion
  using offset_t        = cub::detail::choose_offset_t ;
```

Reply:
Because I only instantiate the benchmark with `int32`, but your suggestion serves the same purpose.